src/spec.h

```diff
@@ -356,6 +356,7 @@ IndexSpecCache *IndexSpec_GetSpecCache(const IndexSpec *spec);
 /**
  * Decrement the reference count of the spec cache. Should be matched
  * with a previous call of GetSpecCache()
+ * Can handle NULL
```

Comment:
why do we need to handle NULL?

Reply:
Instead of checking if it’s null before calling this